person.rb

```diff
@@ -39,6 +44,62 @@ def correct_name()
   end
 end
 
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    # Set the classroom for the student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+
+  def classrooms=(classroom)
+    @classroom&.students&.delete(self)
+    @classroom = classroom
+    classroom.students << self
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] As mentioned by the previous reviewer, you've successfully created the belongs-to side of Rental and Person (a rental belongs to a Person), which is great! However, there's also another requirement in the project to create the belongs-to side of Rental and Book(a rental belongs to a Book), and currently, this part is missing. Please consider adding the relationship function, as mentioned in the previous review.

- [ ] As per the project requirement, the relationship should go both ways and currently, it doesn't. To fix this, you need to add the person to the book as you did for the book to the person. To manage both sides of the relationship in both cases, you'll need to modify your implementations of `add_rentals` in the `Book` class as well. 

This ensures that the relationship is properly established between rentals, books, and persons, meeting the project requirements. Keep up the good work! 👍📚👤

Reply:
Please can you explain what you mean on the second comment.

Reply:
Sorry for the delayed response. I just tried to explain, there is only one requirement and you fulfilled it perfectly. 👍

Reply:
could this be what you want me to do?

```
class Book
  attr_accessor :title, :author
  attr_reader :rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end

  def add_rentals(date, person)
    Rental.new(date, self, person)
  end
end
```